proto/importpb.proto

```diff
@@ -124,8 +124,10 @@ message SSTMeta {
 }
 
 message UploadRequest {
-    SSTMeta meta = 1;
-    bytes data = 2;
+    oneof chunk {
+        SSTMeta meta = 1;
```

Comment:
Do we need to update tags?

Reply:
Not necessary, they are compatible.

Reply:
It doesn't compatible with old version, does it matter?

Reply:
1. It is actually compatible with the previous version.
2. It doesn't matter even if it does not compatible because no one use this yet.